### PR TITLE
Describe data and query options for td operator

### PR DESCRIPTION
**Pull request by nurse**

### CODE REVIEW

File: digdag-docs/src/operators/td.md [hunk 1]
```diff
@@ -52,6 +60,28 @@ When you set those parameters, use [digdag secrets command](https://docs.digdag.
...
+
+* **query**: query
+
+  A query. This string don't expand `${...}` syntax.
+
+  Examples:
+
+  ```
+  td>:
+  data: "SELECT * FROM nasdaq"
```
Comment (yui-knk) [inline digdag-docs/src/operators/td.md]:
```suggestion
  query: "SELECT * FROM nasdaq"
```

File: digdag-docs/src/operators/td.md [hunk 2]
```diff
@@ -9,6 +9,14 @@
     +simple_query:
       td>: queries/simple_query.sql
 
+    +simple_query_expanded:
+      td>:
+      data: "SELECT '${session_id}' FROM nasdaq"
+
```
Comment (yoyama) [inline digdag-docs/src/operators/td.md]: I tried to do this **data** parameter with v0.9.39. The error happened as follows.
```
2019-08-09 19:09:20 +0900 [ERROR] (0041@[0:yy_td]+test1+task1): Configuration error at task +test1+task1: Parameter 'query' is required but not set (config)

```

File: digdag-docs/src/operators/td.md [hunk 3]
```diff
@@ -52,6 +60,28 @@ When you set those parameters, use [digdag secrets command](https://docs.digdag.
...
+  A query. This string don't expand `${...}` syntax.
+
+  Examples:
+
+  ```
+  td>:
+  query: "SELECT * FROM nasdaq"
+  ```
+
```
Comment (yoyama) [inline digdag-docs/src/operators/td.md]: **query** parameter may expand variable as follows.
```
+task1:
  td>:
  query: "SELECT '${session_id}' FROM www_access limit 10"
```
log:
```
2019-08-09 19:11:53 +0900 [INFO] (0051@[0:yy_td]+test1+task1): Started presto job id=...:
SELECT '970' FROM www_access limit 10
```

File: digdag-docs/src/operators/td.md (hunk 2 above)
Comment (nurse) [reply digdag-docs/src/operators/td.md]: `data:` needs an indent and the difference between `query:` was wrong. I updated the description but how to express the difference is little difficult...

File: digdag-docs/src/operators/td.md (hunk 3 above)
Comment (nurse) [reply digdag-docs/src/operators/td.md]: I updated the description.

**Comment by nurse** [comment]
I'm considering more evil examples...
```diff
diff --git a/digdag-docs/src/operators/td.md b/digdag-docs/src/operators/td.md
index 5e76801070..1e6bddb391 100644
--- a/digdag-docs/src/operators/td.md
+++ b/digdag-docs/src/operators/td.md
@@ -78,34 +78,31 @@ When you set those parameters, use [digdag secrets command](https://docs.digdag.
   Examples:
 
-  ```
+  ```
+  # Activate some customers to mail
   _export:
-    database: www_access
-    sql: "SELECT '${task_name\x7D'"
+    database: project_a
 
+  # This API returns following JSON, which has comments to track Presto Jobs and Workflows.
+  # It contains a query whose condition is unfortunately dynamically decided...
+  # ("dynamicaly" means it's not decided when they pushes this workflow)
+  # {"query":"-- https://console.treasuredata.com/app/workflows/sessions/${session_id}\n-- ${task_name}\nSELECT * FROM customers WHERE ..."}
+  +get_queries
+    http>: http://example.com/get_queries
+    store_content: true
+
+  # doesn't expand given string
   +td_data:
     td>:
       data: ${sql}
+    result_connection: mail_something
 
+  # it expands given string
   +td_query:
     td>:
     query: ${sql}
-  ```
+    result_connection: mail_something
+  ```
 
-  Result:
-
-  ```
-  [INFO] (0018@[0:default]+foo+td_data): td>: {data=SELECT '${task_name}'}
-  [INFO] (0018@[0:default]+foo+td_data): td>: {data=SELECT '${task_name}'}
-  [INFO] (0018@[0:default]+foo+td_data): Started presto job id=536633133: SELECT '${task_name}'
-  [INFO] (0018@[0:default]+foo+td_data): td>: {data=SELECT '${task_name}'}
-  [INFO] (0018@[0:default]+foo+td_query): td>:
-  [INFO] (0018@[0:default]+foo+td_query): td-client version: 0.8.11
-  [INFO] (0018@[0:default]+foo+td_query): td>:
-  [INFO] (0018@[0:default]+foo+td_query): Started presto job id=536633088: SELECT '+foo+td_query'
-  [INFO] (0018@[0:default]+foo+td_query): td>:
-  ```
-
-
 * **create_table**: NAME
 
   Name of a table to create from the results. This option deletes the table if it already exists.

```

**Comment by nurse** [comment]
`\x7D` is expected character to avoid expand variables in parsing YAML.

Anyway I introduced more real world example to express the benefit of `query:` option.

File: digdag-docs/src/operators/td.md (hunk 2 above)
Comment (yoyama) [reply digdag-docs/src/operators/td.md]: I confirmed it. Thanks!

**Comment by yoyama** [comment]
With the following workflow, I confirmed the description.

```
_export:
  content: '{ "query": "SELECT ${session_id}, * FROM www_access limit 10" }'
  td:
    database: "sample_datasets"
+task1:
  td>: 
  query: ${JSON.parse(content)["query"]}
```

${session_id} is replaced to _session id_.
But if I changed _query_ to  _data_, it is not replaced.

File: digdag-docs/src/operators/td.md [hunk 4]
```diff
@@ -52,6 +60,49 @@ When you set those parameters, use [digdag secrets command](https://docs.digdag.
   td>: queries/step1.sql
   ```
 
+** **data**: query
```
Comment (yoyama) [inline digdag-docs/src/operators/td.md]: ** is wrong? May be *.
Please check preview page.

**Pull request merged**